pkgs/by-name/cl/clever-tools/package.nix

```diff
@@ -27,8 +27,8 @@ buildNpmPackage rec {
 
   postInstall = ''
```

Comment:
This will probably break for cross-compilation; could you add `lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform)` (the preferred solution may change https://github.com/NixOS/nixpkgs/issues/308283, but this is the most common pattern for now)?

Reply:
Didn't know that but it makes sense, thanks!